scripts/start-rc.bat

```diff
@@ -26,7 +26,7 @@ if exist "hazelcast-remote-controller-%HAZELCAST_RC_VERSION%.jar" (
     echo "hazelcast-remote-controller-%HAZELCAST_RC_VERSION%.jar already exist, not downloading from maven."
 ) else (
     echo "Downloading: remote-controller jar com.hazelcast:hazelcast-remote-controller:%HAZELCAST_RC_VERSION%"
-    call mvn -q dependency:get -Dtransitive=false -DremoteRepositories=%SNAPSHOT_REPO% -Dartifact=com.hazelcast:hazelcast-remote-controller:%HAZELCAST_RC_VERSION% -Ddest=hazelcast-remote-controller-%HAZELCAST_RC_VERSION%.jar || (
+    call mvn -q org.apache.maven.plugins:maven-dependency-plugin:2.10:get -Dtransitive=false -DremoteRepositories=%SNAPSHOT_REPO% -Dartifact=com.hazelcast:hazelcast-remote-controller:%HAZELCAST_RC_VERSION% -Ddest=hazelcast-remote-controller-%HAZELCAST_RC_VERSION%.jar || (
```

Comment:
I see in the PR description `Currently implicitly uses the version bundled with Maven - but this has broken following Maven upgrade`
Whats the actual problem using `maven-dependency-plugin:latest` ??

Reply:
> Whats the actual problem using `maven-dependency-plugin:latest` ??

No longer supports the `-Ddest` parameter, just `get`s the dependency.
The existing `dependency:copy` goal no longer seems to support custom repositories - so even if you already have the EE JAR in your local repository, it refuses to copy it because it can't find it remotely.

Reply:
understood.